### ISSUE TITLE
Deprecated `hashicorp/template` provider

### ISSUE THREAD

**Issue by oponomarov-tu** [1debit/alternat#42]
The last release has introduced [`data "template_cloudinit_config"` ](https://github.com/1debit/alternat/commit/1e594ce2a86a42130fd9d45f3abecca640fc470a#diff-8456b184ddaf930917565415947183b0a96b70d00283e3ab85889359de56c4e7R172), which relies on [`hashicorp/template`](https://registry.terraform.io/providers/hashicorp/template/2.2.0) marked by HashiCorp as deprecated. The provider is no longer built and published for various architecture, like arm for instance. People trying to excecute terraform on M1 will see:

```hcl
│ Error: Incompatible provider version
│
│ Provider registry.terraform.io/hashicorp/template v2.2.0 does not have a package available for your current platform, darwin_arm64.
│
│ Provider releases are separate from Terraform CLI releases, so not all providers are available for all platforms. Other versions of
│ this provider may have different platforms supported.
```

**Comment by bwhaley** [comment]
Doh 🙈 I really shouldn't have made that mistake since I did some work to replace the template provider in a large terraform repo recently. I'll blame it on muscle memory.

A fixed version is in the [`fix-template-provider`](https://github.com/1debit/alternat/tree/fix-template-provider) branch if you want to try it. I'll work to get it merged but it might be a while due to the holidays.

Thanks for filing the issue.

**Comment by bwhaley** [comment]
Released in [v0.2.1](https://github.com/1debit/alternat/releases/tag/v0.2.1).
